arch/arm/src/armv7-a/arm_cpustart.c

```diff
@@ -94,6 +118,10 @@ int arm_start_handler(int irq, FAR void *context, FAR void *arg)
 
   sinfo("CPU%d Started\n", this_cpu());
 
+  /* Set IRQ mode stack for this CPU */
+
+  set_irqstack((uintptr_t)&g_irqtmp + (8  * this_cpu()));
```

Comment:
why not move to arm_vectors.S

Reply:
@xiaoxiang781216 

Because it's difficult to write the same thing in assembler at the beginning of arm_vectorirq().

Reply:
I will study the code tonight to find wether can fix the bug in assemble code, will give the feedback tomorrow.